### PR TITLE
mzamazal doesn't accept new Outreachy candidates anymore

### PR DESCRIPTION
**Pull request by mz-pdm**
Changes proposed in this pull request:

- Let Outreachy candidates know that no new candidates are accepted for mzamazal's projects.

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @mz-pdm

### CODE REVIEW

File: source/community/activities/outreachy.html.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Please find the mentors and projects listed below. In case you are interested in
 
 **Contact Info:** mail: mzamazal@redhat.com, irc: mzamazal in #ovirt channel in irc.oftc.net
 
+I'm sorry, I no longer accept new candidates.
```
Comment (mykaul) [inline source/community/activities/outreachy.html.md]: Perhaps refer them to someone else?

Comment (mz-pdm) [inline source/community/activities/outreachy.html.md]: You mean adding something like "you can still chose from the other oVirt projects"?

Comment (mykaul) [inline source/community/activities/outreachy.html.md]: or contact the devel list.

Comment (mz-pdm) [inline source/community/activities/outreachy.html.md]: Thank you for the suggestion, done.

**Pull request merged**